notebooks/02 EDA.ipynb

```diff
@@ -538,10 +594,20 @@
    },
    "outputs": [],
    "source": [
-    "# check the previous entry to confirm it has a missing \"presenting complaint\"\n",
-    "datetime_df.iloc[datetime_df.presenting_complaint.first_valid_index() - 1][\n",
-    "    [\"START_DATE_TIME_HOSPITAL_PROVIDER_SPELL\", \"presenting_complaint\"]\n",
-    "]"
+    "datetime_df.iloc[109500:110000].wait_minutes"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "nteract": {
+     "transient": {
+      "deleting": false
+     }
+    }
+   },
+   "source": [
+    "After inspection by eye (plot) and by hand (table) we define index `109894` as the start of \"good quality data\", based on the presence of `wait_minutes` data which is indicative of the missing columns at the start of the dataframe."
```

Comment:
It would be useful to have a list of fields which are missing alongside 'wait_minutes"